operations/helm/charts/mimir-distributed/RELEASE.md

```diff
@@ -19,80 +19,142 @@ Weekly releases have the version `x.y.z-weekly.w`, for example `3.1.0-weekly.196
+   - Go back to step 5. Update versions and links in the user/update-mimir-distributed-release-x.y branch to update the next release candidate.
+
+The [release process](https://github.com/grafana/mimir/blob/main/.github/workflows/helm-release.yaml) checks and creates a Git tag formatted as `mimir-distributed-<version>`, for example `mimir-distributed-4.5.0`, on the merge commit created when the PR is merged. To prevent releasing the same version with different content, the release process fails if the tag already exists. The release is published in the [Grafana helm-charts](https://grafana.github.io/helm-charts/) Helm repository.
+
+## Release process for a final release
+
+1. Determine the Helm chart version number.
+
+   [The Chart.yaml file](https://helm.sh/docs/topics/charts/#the-chartyaml-file) requires semantic versioning:
+
+   - The final version has the version `x.y.z`, for example `3.1.0`.
+   - Normally we will proceed the same `x.y.z` version value from the release candidate step.
 
-   For example, `4.5.0`.
+1. Create a branch from release branch to update Mimir/GEM image and helm chart version.
 
-   > **Note:** This is the step that triggers the release process GitHub Action.
+   For example `user/update-mimir-distributed-release-x.y-final`.
 
-1. Set the `appVersion` field, in the [Chart.yaml](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/Chart.yaml) file, to the version of Mimir that the Helm chart deploys.
+1. Update versions in the `user/update-mimir-distributed-release-x.y-final` branch.
 
-   For example, `2.6.0`.
+   - Set the `version` field, in the [Chart.yaml](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/Chart.yaml) file, to the desired final release version.
 
-1. Create or update the release notes in [release notes](https://github.com/grafana/mimir/tree/mimir-distributed-release-4.2/docs/sources/helm-charts/mimir-distributed/release-notes).
+     For example, `4.5.0`.
 
-   The release notes should refer to the correct Mimir and GEM versions and their specific documentation version.
+     > **Note:** Once this change is merged to `mimir-distributed-x.y` branch, it will trigger the release process GitHub Action.
 
-1. Update the Mimir and GEM documentation version parameters in [\_index.md](https://github.com/grafana/mimir/blob/main/docs/sources/helm-charts/mimir-distributed/_index.md)
+   - There shouldn't be anymore update needed in release notes and documentation because that has been done in the release candidate step above.
 
-   The two parameters are `mimir_docs_version` and `gem_docs_version`. With the exception of the release notes, the Helm chart documentation should refer to the documentation or Mimir and GEM that is actually included in the Helm chart.
+1. Open PR to release branch
```

Comment:
Add step above to run `make doc`  (as the chart version is put into the README.md)